api/src/routers/channel-info/index.ts

```diff
@@ -47,7 +47,31 @@ export const channelInfoRouter = Router();
  *       in: query
  *       required: false
  *       schema: 
- *         $ref: "#/components/schemas/ChannelInfoSearchSchema"
+ *         type: object
+ *         properties:
+ *           author:
+ *             type: string
+ *           'topic-type':
+ *             type: string
+ *           'topic-source':
+ *             type: string
+ *           created:
+ *             type: string
+ *             format: date-time
+ *           'latest-message':
+ *             type: string
+ *           limit:
+ *             type: number
+ *           index:
+ *             type: number
+ *         example:
+ *           author: null
+ *           'topic-type': example-channel-data
+ *           'topic-source': null
+ *           created: null
+ *           'latest-message': null
+ *           limit: null
+ *           index: null
```

Comment:
if possible it might be better to use undefined instead of null here

Reply:
To my knowledge undefined is only used in js. This is comment interpreted as yaml (json) so null is the value used for empty fields.